src/util/svg-text-bubble.js

```diff
@@ -133,30 +145,28 @@ class SVGTextBubble {
 
 
     _getTextSize () {
-        const svgString = this._wrapSvgFragment(this._textFragment());
+        const svgString = this._wrapSvgFragment(this._textFragment);
         if (!this._textSizeCache[svgString]) {
             this._textSizeCache[svgString] = this.svgRenderer.measure(svgString);
         }
         return this._textSizeCache[svgString];
     }
 
     _wrapSvgFragment (fragment) {
+        // @todo generate view box
```

Comment:
Why?

Reply:
Throwing away any existing viewbox and generating a new one is part of quirks mode to fix svgs from scratch 2.0. If it's not too hard, speech bubbles should generate their own view boxes, instead of depending on quirks mode (right now I have to make an exception in svgRenderer so that the view box code still gets run on speech bubbles)

Reply:
I added the viewbox to the svg